### ISSUE TITLE
Game entered my guess twice

### ISSUE THREAD

**Issue by jervaters** [psidex/sedordle#3]
While playing today I notice it guessed CHILL twice, when I only entered it once. I had counted to ensure I had one wrong guess left if I wanted to get 16/16 but now only have 9 guesses left. I thought I counted wrong the other day but its possible the same thing could have happened.

![image](https://user-images.githubusercontent.com/17419349/158801390-a4b6c44f-39e6-4b0d-bd40-71a3996f2a1d.png)

**Comment by psidex** [comment]
Yeah this is a bug, as far as I know it only happens if you press enter twice quickly, I could be wrong though.

I'll have to look into it.

Thanks for reporting!
